### ISSUE TITLE
Add preference to choose header line

### ISSUE THREAD

**Issue by brccabral** [mechatroner/vscode_rainbow_csv#104]
Hi,
In some of my CSV files the first couple lines are just information about the file, like author, version, source.
The header line comes only at line 5 or 6.
It would be nice if I could set the actual header line.
Thanks,
Bruno

**Comment by mechatroner** [comment]
Hi, have you tried to use SetHeaderLine command https://github.com/mechatroner/vscode_rainbow_csv#setheaderline ?
Does it fit your use case?

**Comment by brccabral** [comment]
Hi, this command only changes the header when I am in the CSV mode but not in "RBQL Console".
The "RBQL Console" has option to use the first line only.

**Comment by mechatroner** [comment]
Yeah, you are right, I will try to add this functionality, and the VirtualHeader feature should also be supported in RBQL.

**Comment by pbarquei** [comment]
It would be great if this could also set the fixed sticky header line (ref.- https://github.com/mechatroner/vscode_rainbow_csv#enabling-fixed-sticky-header-line ).
I.e., if the header line (with the column names) comes only at line 5, it would be lovely if we could specify it not only to adjust column names, but at the same time set it as the sticky header line, so that scrolling down keeps that line as the fixed header.
